app/components/Search/Search.css

```diff
@@ -49,18 +48,17 @@
 .resultsContainer {
   display: flex;
   flex-wrap: wrap;
-  gap: 2rem;
   max-height: calc(100vh - 120px);
   overflow-y: auto;
-  padding: 2rem 2rem 4rem;
+  padding: var(--spacing-xl) 3rem 3rem;
```

Comment:
Some very few places I've used `3rem` - not ideal and I'll figure out that later

Reply:
Any reason not to add `--spacing-2xl` and so on if you want 3rem? Might be outlined in the article you linked, but I had to create an account to read all of it so I didn't bother

Reply:
he he no. My thought process was that we shouldn't ever really "need" `3rem`, so I figured I should find something else, but I'll probably end up incrementing the `--spacing` further like you suggest. The article only suggest to increment with `8` so `3rem` is definitely "allowed" - just not a fan of different spaces here and there

Reply:
Ah yeah, good point. And I guess it's now an option to stick to -xl as the biggest for a while and then see if we really need 3rem or if we just want to decrease those places to 2rem

Reply:
Yes, exactly - I want to wait a little and see considering I've barely scratched the surface of the webapp here